File: FoodieAppBE/SellerServiceApp/src/main/java/com/project/SellerServiceApp/domain/Order.java
package com.project.SellerServiceApp.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;
import java.util.Map;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Order
{
private String orderId;
private String UserId;
private String restaurantId,restaurantName;
private String orderedOn,orderStatus;
private List<CartItem> CartItems;
private int totalAmount;
private Address address;
}

File: FoodieAppFE/src/app/model/Order.ts
import { CartItems } from "./CartItems";
import { FoodItems } from "./FoodItems";
import { UAddress } from "./User/UAddress";

export class Order{
    orderId?:string;
    userId?:string;
    restaurantId?:string;
    restaurantName?:string;
    orderedOn?:string;
    orderStatus?:string;
    cartItems?:CartItems[];
    totalAmount?:number;
    address?:UAddress; 
}

